src/cosmos/info/thorchainruneInfo.ts

```diff
@@ -98,7 +29,6 @@ const networkInfo: CosmosNetworkInfo = {
 
 const currencyInfo: EdgeCurrencyInfo = {
   currencyCode: 'RUNE',
-  customTokenTemplate: cosmosCustomTokenTemplate,
```

Comment:
I would comment this one out instead of deleting it. There are a few other networks where we don't have tokens, even though the broader family supports it. In all those cases, I left this line as a comment.

Reply:
good point, re-added as comment